example/android/app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,20 @@
     <uses-permission android:name="android.permission.BLUETOOTH" />
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
 
+      <!-- Needed only if your app looks for Bluetooth devices.
+        You must add an attribute to this permission, or declare the
+        ACCESS_FINE_LOCATION permission, depending on the results when you
+      check location usage in your app. -->
+    <uses-permission android:name="android.permission.BLUETOOTH_SCAN" />
```

Comment:
this is only needed for Android12 and currently the underlying repo doesn't't support the new permissions. Can you remove these?